src/plugins/status_page/public/status_page.js

```diff
@@ -28,9 +28,11 @@ const chrome = require('ui/chrome')
 
       const data = resp.data;
       ui.metrics = data.metrics;
-      ui.statuses = data.status.statuses;
       ui.name = data.name;
 
+      ui.statuses = _.groupBy(data.status.statuses, s => s.plugin ? 'plugins' : 'generic');
```

Comment:
This is moot now, based on the discussion in https://github.com/elastic/kibana/pull/7459/files#r67094458